spec/lib/jobs/node-discovered-alert-job-spec.js

```diff
@@ -47,6 +49,8 @@ describe("Job.Alert.Node.Discovered", function () {
         it("should _run() pass without additional data", function() {
             this.sandbox.stub(waterline.nodes, 'needByIdentifier').resolves({ type: 'compute' });
             this.sandbox.stub(eventsProtocol, 'publishNodeEvent').resolves();
+            this.sandbox.stub(lookupService, 'nodeIdToIpAddresses').returns("asdfagagafgdgdgdg");
```

Comment:
`nodeIdToIpAddress` is designed to return a promise, so you need to stub it return promise as well.